tcks/microprofile-opentracing/base/pom.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-tck-microprofile-opentracing</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+
+    <artifactId>quarkus-tck-microprofile-opentracing-base</artifactId>
+    <name>Quarkus - TCK - MicroProfile OpenTracing - Base</name>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>${project.basedir}/src/test/resources/tck-suite.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                    <systemPropertyVariables>
+                        <!-- Disable quarkus optimization -->
+                        <quarkus.arc.remove-unused-beans>false</quarkus.arc.remove-unused-beans>
+                    </systemPropertyVariables>
+                    <!-- This workaround allows us to run a single test using
+                        the "test" system property -->
+                    <!-- https://issues.apache.org/jira/browse/SUREFIRE-569 -->
+                    <dependenciesToScan>
+                        <dependency>org.eclipse.microprofile.opentracing:microprofile-opentracing-tck</dependency>
+                    </dependenciesToScan>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arquillian</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-opentracing</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jackson</artifactId>
```

Comment:
Any reason it's `jackson` instead of `jsonb`?

Reply:
I'm seeing assertion errors from comparing equality of JSON documents after doing the switch from `jackson` to `jsonb`, is `jsonb` preferred and should I dig deeper to see if that can be fixed?

Reply:
It's that MP has a dependency on JSONB and not Jackson, so ideally it should work with JSONB.

@pavolloffay, any reason the TCK shouldn't work with JSONB?

Reply:
The TCK itself depends on jackson https://github.com/eclipse/microprofile-opentracing/blob/1.3.1/tck/base/pom.xml#L75

Reply:
Ok so I dug deeper into it and found the root cause why switching to `jsonb` causes assertion errors (because normally it should still work nevertheless) - it's because with JSONB the tag `http.status_code` is decoded on the client as a `BigDecimal`, with Jackson it is an `Integer`, and the logic in the TCK (https://github.com/eclipse/microprofile-opentracing/blob/1.3.1/tck/base/src/main/java/org/eclipse/microprofile/opentracing/tck/tracer/TestSpan.java#L382) simply determines equality of tags using `Map.equals()`, so it fails with JSONB because one tag has a value of an unexpected type. So if we want to use JSONB we would need to implement a more lenient equality check in the TCK

Reply:
@jmartisk can you raise an issue with MP OT to fix that, as it shouldn't be relying on the presence of Jackson?

Reply:
Yeah, filed https://github.com/eclipse/microprofile-opentracing/issues/159